src/Nethermind/Nethermind.Blockchain/Processing/BlockchainProcessor.cs

```diff
@@ -252,10 +254,7 @@ private void RunProcessingLoop()
             _stats.Start();
             if (_logger.IsDebug) _logger.Debug($"Starting block processor - {_blockQueue.Count} blocks waiting in the queue.");
 
-            if (IsEmpty)
-            {
-                ProcessingQueueEmpty?.Invoke(this, EventArgs.Empty);
-            }
+            FireProcessingQueueEmpty();
```

Comment:
FireIfQueueEmpty?

Reply:
The event is called 'ProcessingQueueEmpty'